### DISCUSSION TITLE
Keycloak reports an error when accessing the login page

### DISCUSSION THREAD

**Discussion by shengwen-xue** [keycloak/keycloak#10349] [Q&A]
When I jump to the keycloak authentication page in other applications, the page reports an error

![image](https://user-images.githubusercontent.com/57387085/154925462-133e4b56-fcbf-4e19-ac9c-d5e2b397920a.png)

![image](https://user-images.githubusercontent.com/57387085/154925669-9df88e97-341f-498a-9402-3a021fc6ede7.png)

17:18:27,984 WARN  [org.keycloak.events] (default task-38) type=LOGIN_ERROR, realmId=aden, clientId=null, userId=null, ipAddress=210.13.90.26, error=invalid_code

keycloak version info: 10.0.1

**Comment by emhug123** [comment]
I’m also experiencing this “userId=null” since last week. Did any of you found an explanation?

I’m getting logs like:

type=LOGIN_ERROR, realmId=XXX, clientId=XXX, userId=null, ipAddress=XXXXXXXXXXX, error=invalid_user_credentials, auth_method=openid-connect, auth_type=code, response_type=code,

keycloak 16

**Reply by dshershov** [reply]
I have same issue with `userId=null`
Did you find any solution? @emhug123

**Reply by shengwen-xue** [reply]
I updated keycloak to 17.0 and I don't have this problem. The configuration of 17.0 is more perfect, you can try

**Comment by buzzo123** [comment]
same problem on keycloak 22

**Comment by webruslanmir** [comment]
+1

**Comment by codespearhead** [comment]
We need the steps to reproduce the issue on our ends.
